katib/katib-controller/overlays/db/kustomization.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
```

Comment:
You need to add `namespace: kubeflow` otherwise they won't be created under the namespace, I think that's the reason DB manager can not be up. @discordianfish

Reply:
@Jeffwan I don't think that's the case, kustomize merely uses these fields to identify what to patch. Most other overlays don't use namespace either.

I'll see if I can disable these 'security checks' to allow basing the ibm-storage overlay on the db overlay to fix the tests, maybe that will also bring the db manager up.